src/System.Private.ServiceModel/tools/scripts/OpenFirewallPorts.cmd

```diff
@@ -0,0 +1,18 @@
+@echo off 
+setlocal
+
+REM 
+REM Start Open all ports WCF test services use
+REM
+
```

Comment:
Just a side note for consideration -- not blocking of this PR.  The existing Bridge self-elevated so that it could do operations like this.  It is possible for the dev today to run from a non-elevated window (using the Bridge) and still succeed for most tests.  We may also shortly add a [ConditionalFact("elevated")] to allow bypassing tests requiring elevation. This current script can only be run elevated, or it will fail (right now silently), which is a loss of functionality. And it also means build.cmd will need to be run elevated too. Personally I believe we want a self-elevating exe to do the 'netsh' work, as we did with the Bridge.  It is also the place where we would parse the ports from command line variables or environment variables, as we did with the Bridge.